modules/log/src/PaymentListBuilder.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\commerce_log;
+
+use Drupal\commerce_payment\PaymentListBuilder as BasePaymentListBuilder;
+
+/**
+ * Overrides the list builder for payments.
+ */
+class PaymentListBuilder extends BasePaymentListBuilder {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function render() {
+    $build = parent::render();
+    $entityIds = [];
+    foreach ($this->load() as $entity) {
+      $entityIds[] = $entity->id();
+    }
+    $build['log']['title'] = [
```

Comment:
Maybe some views expert will figure out how to print the title. It didn't want to. So I punted and did this hack.